pkg/graph/resolvers/trb_request.go

```diff
@@ -67,11 +65,134 @@ func GetTRBRequestByID(ctx context.Context, id uuid.UUID, store *storage.Store)
 
 // GetTRBRequests returns all TRB Requests
 func GetTRBRequests(ctx context.Context, archived bool, store *storage.Store) ([]*models.TRBRequest, error) {
-
 	TRBRequests, err := store.GetTRBRequests(appcontext.ZLogger(ctx), archived)
 	if err != nil {
 		return nil, err
 	}
 	return TRBRequests, err
+}
+
+// UpdateTRBRequestConsultMeetingTime sets the TRB consult meeting time and sends the related emails
+func UpdateTRBRequestConsultMeetingTime(
+	ctx context.Context,
+	store *storage.Store,
+	emailClient *email.Client,
+	fetchUserInfo func(context.Context, string) (*models.UserInfo, error),
+	fetchUserInfos func(context.Context, []string) ([]*models.UserInfo, error),
+	id uuid.UUID,
+	meetingTime time.Time,
+	copyTRBMailbox bool,
+	notifyEUAIDs []string,
+	notes string,
+) (*models.TRBRequest, error) {
+	notifyInfos, err := fetchUserInfos(ctx, notifyEUAIDs)
```

Comment:
If `notifyEUAIDs` is empty, this function will return an error; probably best to guard the call to `fetchUserInfos` with a `len(notifyEUAIDs) == 0` check, otherwise set `notifyInfos` to an empty slice (or guard the setup to `notifyEmails` as well, and make sure the later initilization of `email.SendTRBRequestConsultMeetingEmailInput` takes an empty slice for `NotifyEmails`.